bigquery/snippets/snippet.go

```diff
@@ -91,6 +91,60 @@ func updateDatasetAccessControl(client *bigquery.Client, datasetID string) error
 	return nil
 }
 
+func getDatasetLabels(client *bigquery.Client, w io.Writer, datasetID string) error {
+	ctx := context.Background()
+	// [START bigquery_get_dataset_labels]
+	meta, err := client.Dataset(datasetID).Metadata(ctx)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(w, "Dataset %s labels:\n", datasetID)
+	if meta.Labels == nil {
+		fmt.Fprintln(w, "dataset has no labels defined.")
+		return nil
+	}
+	for k, v := range meta.Labels {
+		fmt.Fprintf(w, "\t%s:%s\n", k, v)
+	}
+	// [END bigquery_get_dataset_labels]
+	return nil
+}
+
+func updateDatasetAddLabel(client *bigquery.Client, datasetID string) error {
+	ctx := context.Background()
+	// [START bigquery_label_dataset]
+	ds := client.Dataset(datasetID)
+	original, err := ds.Metadata(ctx)
```

Comment:
The variable names for the original and the update are inconsistent between the functions. Maybe they could be `meta` and `update`?

Reply:
unified naming, including older samples.